go/observability/observability.go

```diff
@@ -0,0 +1,29 @@
+package observability
+
+import (
+	"log"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/trace"
+)
+
+func ConfigureStackdriver(gcpProject string) {
+	if err := view.Register(ocgrpc.DefaultClientViews...); err != nil {
+		log.Fatalf("Failed to register ocgrpc client views: %v", err)
+	}
+
+	sd, err := stackdriver.NewExporter(stackdriver.Options{
```

Comment:
The exporter isn't returned and used by the clients/servers. How does opencensus know where to send the data?

Is everything configured globally, only one project can be used, and all data will be send to this one project?

Reply:
Yes, it's all global within the process. That's OpenCensus's model (in all languages)